components/iap-enabler/README.md

```diff
@@ -0,0 +1,10 @@
+# IAP Enabler
+
+IAP Enabler is a docker image which is used to enable iap. We build this and push it to gcr so that it can be used in private GKE clusters
```

Comment:
I think we use BackendConfig now to enable IAP. I think this is only used to set the timeout on the loadbalancer.

Reply:
I'll rebase this once https://github.com/kubeflow/kubeflow/pull/1327 gets merged